lib/views/home_page.dart

```diff
@@ -0,0 +1,147 @@
+import 'dart:async';
+
+import 'package:concordi_around/views/go_to_page.dart';
+import 'package:concordi_around/widgets/drawer.dart';
+import 'package:concordi_around/widgets/search/main_search_bar.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:geolocator/geolocator.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+class HomePage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return _HomePageState();
+  }
+}
+
+class _HomePageState extends State<HomePage> {
+
+  Completer<GoogleMapController> _controller = Completer();
+  Geolocator _geolocator;
+  Position _position;
+  CameraPosition _cameraPosition;
+  StreamSubscription _positionStream;
+
+  @override
+  Widget build(BuildContext context) {
+
+  if (_cameraPosition == null) {
+      _cameraPosition = CameraPosition(target: LatLng(0, 0));
+  }
+
+    return Scaffold(
+        body: Stack(
+              children: <Widget>[
+                // Replace this container with your Map widget
```

Comment:
What is your Map widget? Is this meant for a reminder?

Reply:
Deleted this comment.